### ISSUE TITLE
Bug in scenario 24?

### ISSUE THREAD

**Issue by mrcreavill** [teamducro/gloomhaven-storyline#142]
Hello,

Just noticed that completing scenario 24 does not unlock scenarios: 32 and 30. Is this a bug? I believe that both 30 and 32 should be unlocked by completing 24.

Thanks in advance

**Comment by dducro** [comment]
Yeah, this is a known issue caused by an update for support of JotL.
I'll investigate and fix it as soon as possible.
